tests/test_merge.py

```diff
@@ -181,6 +181,43 @@
             id="inter:node_over_node_interpolation",
         ),
         # Structured configs
+        pytest.param(
+            (DictConfig({}, element_type=User), {"user007": {"age": 99}}),
+            {"user007": {"name": "???", "age": 99}},
+            id="dict:merge_into_sc_element_type:expanding_new_element",
+        ),
+        pytest.param(
+            (
+                DictConfig({"user007": "???"}, element_type=User),
+                {"user007": {"age": 99}},
+            ),
+            {"user007": {"name": "???", "age": 99}},
+            id="dict:merge_into_sc_element_type:into_missing_element",
+        ),
+        pytest.param(
+            (
+                DictConfig({"user007": User("bond", 7)}, element_type=User),
+                {"user007": {"age": 99}},
+            ),
+            {"user007": {"name": "bond", "age": 99}},
+            id="dict:merge_into_sc_element_type:merging_with_existing_element",
+        ),
+        pytest.param(
+            (
+                DictConfig({"user007": "???"}, element_type=User),
+                {"user007": {"age": 99}},
+            ),
+            {"user007": {"name": "???", "age": 99}},
+            id="dict:merge_into_sc_element_type:merging_into_missing",
+        ),
```

Comment:
I think that the tests with `id="dict:merge_into_sc_element_type:into_missing_element"` and `id="dict:merge_into_sc_element_type:merging_into_missing"` are duplicate.

Reply:
good catch